apps/src/applab/DesignModeBox.jsx

```diff
@@ -1,6 +1,5 @@
 /* global $ */
```

Comment:
I'm willing to listen to arguments as to why that is a better approach than just saying it's always an allowable global. One pretty easy argument to make is that whichever approach we take, React and $ should be the same. What do you think?